app/src/userpages/components/Avatar/index.jsx

```diff
@@ -0,0 +1,84 @@
+// @flow
+
+import React from 'react'
+import { Button } from 'reactstrap'
+import cx from 'classnames'
+import { Translate } from 'react-redux-i18n'
+
+import FileUpload from '$shared/components/FileUpload'
+import styles from './avatar.pcss'
+
+type NameAndEmailProps = {
+    name?: string,
+    email?: string,
+}
+
+type UploadProps = {
+    image: string,
+}
+
+type Props = NameAndEmailProps & UploadProps & {
+    editable?: boolean,
+    className?: string,
+}
+
+const NameAndEmail = ({ name, email }: NameAndEmailProps) => (
```

Comment:
Maybe this one and `AvatarUpload` could be extracted into their own files along with their styling and stuff?